extension/storage/local/filestorage/default_others.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+package filestorage
+
+// TODO do we like these path names?
+func getDefaultDirectory() string {
+	return "/var/lib/otelcol/file_storage"
```

Comment:
What about:
```suggestion
	return "/var/lib/otelcol/data"
```
Do you want to see the extension name in the path?

Reply:
I don't have a strong opinion on this one, but one possible reason to indicate the extension type in the path is to allow for a degree of isolation from alternate implementations.

Perhaps a file naming extension would protect against this. (i.e. `/var/lib/otelcol/data/kind_type_name.otdb`)

Reply:
Actually, I think what you have is better. Let's name the directory after the extension.